doc-Infrastructure_Migration_Solution_Guide/cfme/doc/modules/proc_Creating_an_ansible_service_catalog_item.adoc

```diff
@@ -0,0 +1,10 @@
+[id="Creating_an_ansible_service_catalog_item"]
+= Creating an Ansible Service Catalog Item
+
+The CloudForms Inventory must contain at least one Ansible repository, one playbook, and one credential.
```

Comment:
1. Replace 'CloudForms' with {product-title} if this doc also exists in the upstream repo so the correct product title is inserted during the build.
 2. Inventory should be lowercase.

Reply:
CloudForms is not the product. IMS is.

Reply:
Yes, it could. We use {product-title} so that the name renders properly depending on where it's published - upstream (ManageIQ) vs downstream (CloudForms). If IMS content is intended to appear in upsteam docs we'll need to make the distinction. Adding an additional variable should do the trick.

Reply:
I removed the only place where I had used {product-title}. Also, I had created ims.adoc in the common > attributes folder, rather than changing the cfme.adoc file. Will this be sufficient to avoid problems down the road? Or should I add a cfme.adoc file in the attributes folder and use {product-title} wherever CF occurs?